commit-msg.sh

```diff
@@ -1,14 +0,0 @@
-#!/bin/bash
-#
-# Solve the problem of [Wishlist: support for ES6 modules. #14].
-# https://github.com/legend80s/commit-msg-linter/issues/14
-
-# This shell script template is from: https://google.github.io/styleguide/shellguide.html
-
-# $0: .git/hooks/commit-msg, $1: .git/COMMIT_EDITMSG, $2: , $3:
-# node '.git/hooks/commit-msg-linter.js' $1
-
-# id=commit-msg-linter - The comment SHOULD NOT BE DELETED.
```

Comment:
`id=commit-msg-linter - The comment SHOULD NOT BE DELETED.`

https://github.com/legend80s/git-commit-msg-linter/blob/7f164f1ec571720249fbcab29c9e420a5c0223cc/uninstall.js#L48

Reply:
The name of `commit-msg-linter.js` file has already included `commit-msg-linter`. Maybe we can delete it this line?

Reply:
We use this id `const ID = 'commit-msg-linter'; ` to check whether the commit-msg file is created by us, if it is then we can remove it confidently on uninstallation. May be we can add it to this line 

```diff
- const rules = `cat ${path.resolve(__dirname, 'commit-msg-linter.js')} | node --input-type=commonjs`
+ const rules = `
+ # id=commit-msg-linter - The comment SHOULD NOT BE DELETED.
+ # It's used to check whether this commit-msg hook file is created by us, if it is then we can remove it confidently on uninstallation.
cat ${path.resolve(__dirname, 'commit-msg-linter.js')} | node --input-type=commonjs
`
```